_data/simple-icons.json

```
@@ -17941,6 +17941,12 @@
             "hex": "009CFB",
             "source": "https://www.zingat.com"
         },
+        {
+            "title": "Zip Pay",
```

```suggestion
            "title": "Zip",
```
The name seems to be just "Zip"? Not seeing "Zip Pay" anywhere but guessing it is one of their products?